docker-compose.client.demo.yml

```diff
@@ -7,3 +7,7 @@ services:
     mender-client:
         ports:
             - "85:85"
+        tmpfs:
+            - /data
+            - /tmp
+            - /mnt
```

Comment:
As little as I understand docker-compose, as this is already in `docker-compose.client.yml` we don't need to copy it here. Let's get advantage of inheritance design of the tool.